examples/affinity/cookie/nginx/sticky-ingress.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: nginx-test
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/affinity: "cookie"
```

Comment:
So re: my previous comment about one annotation, if just `ingress.kubernetes.io/affinity` is supplied and nothing else, we still get sticky with default alg and default cookie name correct?